tests/test_models/test_inference/test_forecast.py

```diff
@@ -459,36 +479,54 @@ def _test_forecast_out_sample_prefix(ts, model, transforms, full_prediction_size
             (DeadlineMovingAverageModel(window=1), []),
             (BATSModel(use_trend=True), []),
             (TBATSModel(use_trend=True), []),
+            (RNNModel(input_size=1, encoder_length=7, decoder_length=7, trainer_params=dict(max_epochs=1)), []),
+        ],
+    )
+    def test_forecast_out_sample_prefix(self, model, transforms, example_tsds):
+        self._test_forecast_out_sample_prefix(example_tsds, model, transforms)
+
+    # if we ignore this warning there is assertion error
```

Comment:
May you create an issue

Reply:
Yes, I think we should investigate why old NNs isn't working as expected. But I don't think we should make the fix inside this task.